test/hotspot/jtreg/runtime/Thread/TestAlwaysPreTouchStacks.java

```diff
@@ -82,6 +72,76 @@ static private final Thread createTestThread(int num) {
         t.setDaemon(true);
```

Comment:
Since you are here, could you add a comment here?

```suggestion
        // Let test threads run as daemons to ensure that they are still running and
        // that their stacks are still allocated when the JVM shuts down and the final
        // NMT report is printed.
        t.setDaemon(true);
```

Reply:
Done.